src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -599,6 +599,9 @@
     <Compile Include="System\Net\Http\HttpClientHandler.Core.cs" />
     <Compile Include="uap\System\Net\HttpClientHandler.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Reference Include="System.Net.NameResolution" />
+  </ItemGroup>
```

Comment:
We didn't already have this because we were relying on the underlying implementation (e.g. System.Net.Sockets) to do the original Dns call?

Reply:
Correct.  I needed to add it in order to compile.